pkg/security/secl/containerutils/cgroup.go

```diff
@@ -24,6 +24,12 @@ const (
 	CGroupManagerSystemd                          // systemd
 )
 
+// CGroup flags
+const (
+	SystemdService CGroupFlags = (0 << 8)
```

Comment:
do we agree that's equal to .. zero ? :)

Reply:
Yes :-)